cms/models.py

```diff
@@ -950,6 +950,16 @@ class Meta:
         blank=True,
         help_text="A short description indicating how long it takes to complete (e.g. '4 weeks')",
     )
+    min_weeks = models.SmallIntegerField(
+        null=True,
+        blank=True,
+        help_text="The minimum number of weeks required to complete the course/program.",
+    )
+    max_weeks = models.SmallIntegerField(
+        null=True,
+        blank=True,
+        help_text="The maximum number of weeks required to complete the course/program.",
+    )
```

Comment:
We also need to make changes in `external_course_sync_api.py` to add the `min_weeks` and `max_weeks` field in the `ExternalCourse` class. Also, in the `create_or_update_external_course_page`, we need to pass the fields in the `ExternalCoursePage` constructor to set the values in an external course. We may also need to update the tests in `external_course_sync_api_test.py`